neurom/fst/_neuritefunc.py

```diff
@@ -315,39 +315,31 @@ def partition_pairs(neurites, neurite_type=NeuriteType.all):
                              neurite_filter=is_type(neurite_type)))
 
 
-def section_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None):
-    '''Remote bifurcation angles in a collection of neurites'''
+def section_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None,
+                             iterator_type=Tree.ipreorder):
+    '''Remote bifurcation angles in a collection of neurites.
+    The iterator_type can be used to select only terminal sections (ileaf)
+    or only bifurcations (ibifurcation_point).'''
     dist = []
     for n in iter_neurites(neurites, filt=is_type(neurite_type)):
         pos = n.root_node.points[0] if origin is None else origin
         dist.extend(sectionfunc.section_radial_distance(s, pos)
-                    for s in n.iter_sections())
-
+                    for s in iter_sections(n,
+                                           iterator_type=iterator_type,
+                                           neurite_filter=is_type(neurite_type)))
```

Comment:
I also just noticed that I use the neurite_type filter twice. I think the first one should be sufficient, right?